integration_tests/agp/build.gradle

```diff
@@ -26,7 +26,7 @@ dependencies {
     testImplementation project(":integration_tests:agp:testsupport")
 
     testImplementation "junit:junit:${junitVersion}"
-    testImplementation("androidx.test:core:$axtVersion")
-    testImplementation("androidx.test:runner:$axtVersion")
+    testImplementation("androidx.test:core:$axtCoreVersion")
```

Comment:
Does it need `@aar` like below?

Reply:
It's ok. I would also like to see if the latest AndroidX test version fixes some flaky tests of Robolectric's instrumentation tests that I have encountered.